content/docs/getting-started/futures.md

```diff
@@ -74,282 +127,105 @@ They cannot because they don't have the actual HTTP response. Instead, they
 define the work to be done when the response future completes.
 
 Both the [`futures`] crate and Tokio come with a collection of combinator
-functions that can be used to work with futures.
+functions that can be used to work with futures. So far we've seen `and_then` which
+chains two [`futures`] together, `then` which allows to chain a future to a previous
+one even if the previous one errored, and `map` which simply maps a future's value
+from one type to another.
```

Comment:
It might be worth adding some code examples w/ some other combinators? Maybe showing select and join?

Reply:
Hmm yea that could be nice, but I don't want this to turn into a guide on combinators. Let me play with it a bit.